ibc/utils.go

```diff
@@ -61,7 +62,7 @@ func GetTransferAmount(packet channeltypes.Packet) (string, error) {
 		return "", errorsmod.Wrapf(errortypes.ErrInvalidCoins, "empty amount")
 	}
 
-	if _, ok := math.NewIntFromString(data.Amount); !ok {
+	if _, ok := sdk.NewIntFromString(data.Amount); !ok {
```

Comment:
please use `sdkmath` moving forward otherwise we'll need to update it again in the next SDK release